graphics/gx/controller/source/controller.c

```diff
@@ -0,0 +1,10871 @@
+#include <gccore.h>
```

Comment:
All of https://github.com/devkitPro/gamecube-examples/pull/1/files#diff-39b8e501e44cc7040794d76b43e9a9225328f3db698ddbafd345bed22024cd83R1-R14 should probably be in the generated header along with externs for the generated arrays.